### DISCUSSION TITLE
Add Breadcrumbs functionality

### DISCUSSION THREAD

**Discussion by amitaibu** [digitallyinduced/ihp#1166] [Ideas]
What do you think of adding a Breadcrumbs functionality a la Pagination. Something like this, that would be rendered through CSSFramework?

```haskell
BreadcrumbsItem = BreadcrumbsItem 
{ label :: Text
, url :: Maybe Text
}

renderBreadcrumbs :: [ BreadcrumbsItem ] -> Bool -> Html
renderBreadcrumbs breadcrumbsItem showFirstItemAsLinkToHomepage
```

**Comment by mpscholten** [comment]
Sounds like a good addition to me 👍

**Comment by mpscholten** [comment]
We should consider if it makes sense to have `label :: Html`.

Then a item label could contain custom formatting if needed or support e.g. SVG icons.
